wp-includes/functions.php

```diff
@@ -5736,19 +5736,36 @@ function wp_validate_boolean( $var ) {
 /**
  * Delete a file
  *
- * @since 4.2.0
+ * @since 4.9.2
  *
- * @param string $file The path to the file to delete.
+ * @param string $file                The path to the file to delete.
+ * @param array  $directories_unblock Are directory to unblocked. They are ROOT, WP_CONTENT_DIR, WP_ADMIN_DIR
+ *                                   and WP_INCLUDE_DIR.
+ * @return bool Whether the param is invalidated.
  */
-function wp_delete_file( $file ) {
+function wp_delete_file( $file, $directories_unblock = array() ) {
```

Comment:
I'm not sure the `$directories_unblock` name param name is descriptive enough. Also it reads weird, so might be better to rename to `$unblock_directories` or something.